quinn-proto/src/connection.rs

```diff
@@ -707,57 +671,49 @@ impl Connection {
         sent_time <= self.recovery_start_time
     }
 
-    fn set_loss_detection_alarm(&mut self) {
+    fn set_loss_detection_timer(&mut self) {
         if self.bytes_in_flight == 0 {
             self.io.timer_stop(Timer::LossDetection);
             return;
         }
 
-        let mut alarm_duration: u64;
-        if self.awaiting_handshake {
+        let mut timeout: u64;
```

Comment:
I'm not sure this makes sense? This `timeout`'s liveness in `if self.crypto_in_flight` seems fully disjoint from the other stuff that uses this declaration below, so it would be clearer to just declare this twice?

Reply:
This is another case of me erring on the side of transcribing slightly weird pseudocode from the draft. Think we should diverge for clarity's sake?

Reply:
Can you point me to the section in the spec?

Reply:
I was thinking of https://tools.ietf.org/html/draft-ietf-quic-recovery-17#section-6.4.7, but on review I see there's no declaration at *all*, so I suppose I might as well scope it sanely.